qcengine/programs/tests/test_programs.py

```diff
@@ -413,3 +413,50 @@ def test_mace_gradient():
     result = qcng.compute(atomic_input, "mace")
     assert result.success
     assert pytest.approx(result.return_result) == expected_result
+
+
+@using("aimnet2")
+@pytest.mark.parametrize(
+    "model, expected_energy",
+    [
+        pytest.param("b973c", -76.39604306960972, id="b973c"),
+        pytest.param("wb97m-d3", -76.47412023758551, id="wb97m-d3"),
+    ],
+)
+def test_aimnet2_energy(model, expected_energy):
+    """Test computing the energies of water with two aimnet2 models."""
+
+    water = qcng.get_molecule("water")
+    atomic_input = AtomicInput(molecule=water, model={"method": model, "basis": None}, driver="energy")
+
+    result = qcng.compute(atomic_input, "aimnet2")
+    assert result.success
+    assert pytest.approx(result.return_result) == expected_energy
+    assert "charges" in result.extras
+    assert "charges_std" in result.extras
+    assert "forces_std" in result.extras
+
+
+@using("aimnet2")
+def test_aimnet2_gradient():
+    """Test computing the gradient of water using one aimnet2 model."""
+
+    water = qcng.get_molecule("water")
+    atomic_input = AtomicInput(molecule=water, model={"method": "wb97m-d3", "basis": None}, driver="gradient")
+
+    result = qcng.compute(atomic_input, "aimnet2")
+    assert result.success
+    # make sure the gradient is now the return result
+    assert np.allclose(
+        result.return_result,
+        np.array(
+            [
+                [-0.0, 2.6080331227973375e-09, -0.04097248986363411],
+                [-0.0, -0.029529934749007225, 0.020486244931817055],
+                [-0.0, 0.029529931023716927, 0.020486244931817055],
```

Comment:
How is AIMNET2 handling orientation for arrays? When Molecule fix_com/fix_orientation=T, qcng harnesses are supposed to return the gradient in the same frame as the Molecule.geometry. https://github.com/MolSSI/QCEngine/blob/master/qcengine/programs/tests/test_alignment.py tests this for a few programs, but an AI model probably can't produce HF/Dunning basis. 

fix_*=False (the default) should return the program's standard orientation (whatever that means for the program).

Reply:
The model should always return the result in the same orientation as the input similar to the torchani interface.